TACC/Core-Styles
# make label checkers run on more specific events

## Overview / Related / Changes

Do not run PR label checker until PR has been manually or automatically labeled.

## Testing

1. Open PR with label set before clicking "Create pull request".
2. Check PR label checker is run.
3. Remove label from PR.
4. Check PR label checker is run.
5. Close & Reopen PR.
6. Check PR label checker is not run.
7. Add label.
8. Check PR label checker is run.

## UI

N/A
